controls/cis_rhel8.yml

```diff
@@ -0,0 +1,2285 @@
+  - id: 1.8.1.3
+    title: Ensure remote login warning banner is configured properly (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  - id: 1.8.1.4
+    title: Ensure permissions on /etc/motd are configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - file_groupowner_etc_motd
+      - file_owner_etc_motd
+      - file_permissions_etc_motd
+
+  - id: 1.8.1.5
+    title: Ensure permissions on /etc/issue are configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - file_groupowner_etc_issue
+      - file_owner_etc_issue
+      - file_permissions_etc_issue
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/7225
+  - id: 1.8.1.6
+    title: Ensure permissions on /etc/issue.net are configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  - id: 1.8.2
+    title: Ensure GDM login banner is configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - dconf_gnome_banner_enabled
+      - dconf_gnome_login_banner_text
+      - login_banner_text=usgcb_default
+
+  - id: 1.9
+    title: Ensure updates, patches, and additional security software are installed (Manual)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+    related_rules:
+      - security_patches_up_to_date
+
+  - id: 1.10
+    title: Ensure system-wide crypto policy is not legacy (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - configure_crypto_policy
+
+  # This rule works in conjunction with the configure_crypto_policy above.
+  # If a system is remediated to CIS Level 1, just the rule above will apply
+  # and will enforce the default value for var_system_crypto_policy (DEFAULT).
+  # If the system is remediated to Level 2 then this rule will be selected,
+  # and the value applied by the rule above will will be overridden to
+  # FUTURE through the var_system_crypto_policy variable.
+  - id: 1.11
+    title: Ensure system-wide crypto policy is FUTURE or FIPS (Automated)
+    levels:
+      - l2_server
+      - l2_workstation
+    automated: yes
+    rules:
+      - var_system_crypto_policy=future
+
+  - id: 2.1.1
+    title: Ensure xinetd is not installed (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - package_xinetd_removed
+
+  - id: 2.2.1.1
+    title: Ensure time synchronization is in use (Manual)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+    related_rules:
+      - package_chrony_installed
+
+  - id: 2.1.1
+    title: Ensure chrony is configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - service_chronyd_enabled
```

Comment:
Why this rule? It is not mentioned in the benchmark.
Moreover, I suggest adding this variable:
var_multiple_time_servers=rhel

Reply:
This is what we're using in the current profile.

https://github.com/ComplianceAsCode/content/blob/21d5188001edba157e5d925e0012a5609d77f00b/products/rhel8/profiles/cis.profile#L271-L274

This is an interesting one though. I agree that, by pure wording, the benchmark doesn't actually call for Chrony to be running, but between 2.2.1.1 calling for Chrony to be _installed_, and this control calling for Chrony to be _configured_, I feel like it's implied that the goal of the benchmark is for the service should actually be running and enabled.

But if we want to be strictly correct as per the benchmark then it doesn't call for this, so we could remove this rule.